docs/wiki/resources.md

```diff
@@ -49,19 +49,18 @@
     [Grin transactions user interactivity](https://forum.grin.mw/t/grin-transactions-user-interactivity/7738)</br>
     [Eliminate finalize step](https://forum.grin.mw/t/eliminating-finalize-step/7621)</br>
     [Integrated payment proofs and round minimization](https://forum.grin.mw/t/integrated-payment-proofs-and-round-minimization/7745)</br>
-    [Pep talk for one sided transactions](https://forum.grin.mw/t/pep-talk-for-one-sided-transactions/7361)</br>
```

Comment:
I think this topic got mangled when Grundkurs requested his account be removed. I un-deleted it to preserve the interesting discussion.

Reply:
I re-added the link